pkg/subsystem/lists/linux_upstream/subsystems.go

```diff
@@ -0,0 +1,4619 @@
+// AUTOGENERATED FILE
+
+package linuxupstream
+
+import . "github.com/google/syzkaller/pkg/subsystem/entity"
+import "github.com/google/syzkaller/pkg/subsystem"
+
+func init() {
+	subsystem.RegisterList("linux_upstream", subsystems)
```

Comment:
I think it's more reasonable to call this just "linux" (and the package). The extraction is in the linux package as well. If we add some downstream variations, we can call them "android" or "linux-android".

Reply:
Wouldn't go then only compile the package if GOOS=linux? 🤔